GitUI/UserControls/RevisionGrid/CopyContextMenuItem.cs

```diff
@@ -62,7 +62,7 @@ private void AddItem([NotNull] string displayText, [NotNull] string textToCopy,
 
             var item = new ToolStripMenuItem
             {
-                Text = displayText,
+                Text = displayText.TrimEnd('\r', '\n'),
```

Comment:
Why is it not needed when invoked from the revision graph?

Reply:
This is a general situation for the main grid and for the left panel. When multiple lines increase the height of the menu item this break the layout. If it is in the middle, it does not look weird than if it is at the end of the text.